docs/install.rst

```diff
@@ -58,7 +58,7 @@ Optional Snappy install
 Install Development Libraries
 =============================
 
-Download and build Snappy from https://google.github.io/snappy/
```

Comment:
This doesn't 404 for me, are you sure it's a bad link?

Reply:
Huh. It's live now. Thanks for double-checking!